services/QuillLMS/db/migrate/20220920190724_drop_old_concept_results.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DropOldConceptResults < ActiveRecord::Migration[6.1]
+  def up
+    remove_column :concept_results, :old_concept_result_id
+    drop_table :old_concept_results
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
1. I would recommend doing this migration in a subsequent commit.  That way you are likely to avoid any issues with stale code referencing a table that is no longer there.

2. In general I try to avoid irreversible migrations even with drop table.  Rails will complain about irreversible migration with methods like `drop_table` and `remove_{column|reference}` etc, but in most cases this is just a matter of adding type information along with reference/index commands:

```
  def change
    remove_reference :concept_results, :old_concept_result_id, :int 
    remove_reference :old_concept_results, :activity_classification, foreign_key: true
    remove_index :old_concept_results, :concept_id

    drop_table :old_concept_results do |t|
      t.integer :activity_session_id
      t.integer :concept_id, null: false
      t.string :question_type
      t.json :metadata
    end
  end
```